app/microserviceshandler.py

```diff
@@ -160,7 +160,7 @@ async def get(self, request):
 
     if 'clear' in microservice_name:
       microservice_name = microservice_name.replace('clear', '')
-      await clear_stored_search_criteria(session, microservice_name)
+      await clear_stored_search_criteria(session, request, microservice_name)
```

Comment:
If we are passing `request`, we do not need to pass `session`

Reply:
Adjusted